pax/plugins/peak_processing/BasicProperties.py

```diff
@@ -86,10 +87,10 @@ def transform_event(self, event):
             # Don't weigh negative samples for computation of center of gravity
             weights = np.clip(w, 0, float('inf'))
             if not np.sum(weights) > 0:
-                self.log.warning("Sum waveform of peak %d-%d (%0.2f pe area) sums to a nonpositive value... unusual!"
-                                 " Cannot align peak's sum waveform, storing zeros instead." % (peak.left,
+                self.log.debug("Sum waveform of peak %d %d-%d (%0.2f pe area) sums to a nonpositive value... unusual!"
+                                 " Cannot align peak's sum waveform, and will be deleted." % (peak_i, peak.left,
```

Comment:
__[Flake8]__

[E127] continuation line over-indented for visual indent


<!-- Comment by [Sider](https://sider.review) -->